tests/rgw/sanity_rgw.py

```diff
@@ -94,6 +97,22 @@ def run(ceph_cluster, **kw):
             sudo=True, cmd=f"yum install -y {pkgs}", long_running=True
         )
 
+    if install_start_kafka:
+        log.info("install and start kafka broker for bucket notification tests")
+        rgw_node.exec_command(
+            cmd="wget http://magna002.ceph.redhat.com/cephci-jenkins/kafka_2.13-2.8.0.tgz && tar -zxvf kafka_2.13-2.8.0.tgz && mv kafka_2.13-2.8.0 kafka && chown cephuser:cephuser kafka", long_running=True
+        )
```

Comment:
This would not work with IBM-Cloud hence, we could do something like
```
cloud_type = kw.get("cloud_type", "openstack")
if cloud_type == "ibmc":
  wget_cmd = "curl -o /tmp/kafka.tgz http://10.245.4.4/kafka_2.13-2.8.0.tgz"
else:
  wget_cmd = "curl -o /tmp/kafka.tgz http://magna002.ceph.redhat.com/cephci-jenkins/kafka_2.13-2.8.0.tgz"

tar_cmd = "tar -zxvf /tmp/kafka.tgz -C /usr/local/kafka"
chown_cmd = "chown cephuser:cephuser /usr/local/kafka"

rgw_node.exec(cmd=f"{wget_cmd} && {tar_cmd} && {chown_cmd}", sudo=True)
```

Reply:
@psathyan , thanks for the input. made the changes suggested